package.json

```diff
@@ -366,7 +366,8 @@
     "socket.io-parser": "4.2.4",
     "wrap-ansi": "7.0.0",
     "webpack-dev-middleware": "5.3.4",
-    "eslint": "8.56.0"
+    "eslint": "8.56.0",
+    "ws": "8.17.1"
```

Comment:
Should we be adding this as a dep, or setting a resolution? Afaik, if we don't use directly, but want a later version, it should be a resolution

Reply:
I couldn't find us using it directly, so I added it as a resolution.

Reply:
My bad, didn't see it was in resolutions

Reply:
No problem. I wasn't certain when you said it so I double checked which is worth it 👍